src/ride/vehicle.c

```diff
@@ -1969,36 +1969,39 @@ static bool vehicle_can_depart_synchronised(rct_vehicle *vehicle)
 	int z = ride->station_heights[station];
 
 	rct_map_element *mapElement = map_get_track_element_at(x, y, z);
+	if (mapElement == NULL) {
+		return false;
```

Comment:
I made it return `false` here, as this seems the safe option, but please verify.